### PR TITLE
update pneuma configuration using keyword executor

### PR DESCRIPTION
**Pull request by khyberspache**

### CODE REVIEW

File: command-and-control/23c5cb2f-3f6f-4770-b260-13edf23f5903.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+id: 23c5cb2f-3f6f-4770-b260-13edf23f5903
+metadata:
+  version: 1
+  authors:
+    - khyberspache
+  tags: []
+name: Update Pneuma configuration
+description: |
+  Pneuma is an open-source agent which accompanies the Prelude platform. This procedure sends an instruction to update the
```
Comment (privateducky) [inline command-and-control/23c5cb2f-3f6f-4770-b260-13edf23f5903.yml:9]: change description to generic (Non-Pneuma) and let's do variations per agent that we have config swapping for (pneuma lol)

File: command-and-control/23c5cb2f-3f6f-4770-b260-13edf23f5903.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  agent's configuration.
+tactic: command-and-control
+technique:
+  id: T1573.001
+  name: "Encrypted Channel: Symmetric Cryptography"
+platforms:
+  windows:
+    keyword:
+      command: 'config.{"Contact": "udp", "Address": "0.0.0.0:4545", "Sleep": 10}'
```
Comment (privateducky) [inline command-and-control/23c5cb2f-3f6f-4770-b260-13edf23f5903.yml:18]: can we remove the ' quotes..? I don't think they're necessary..?

**Review by privateducky — CHANGES_REQUESTED** [review]
small things

File: command-and-control/23c5cb2f-3f6f-4770-b260-13edf23f5903.yml (hunk 2 above)
Comment (khyberspache) [reply command-and-control/23c5cb2f-3f6f-4770-b260-13edf23f5903.yml:18]: They are necessary otherwise yaml breaks

**Review by privateducky — APPROVED** [review]

**Pull request merged**